### DISCUSSION TITLE
ФРМО. Пример заполнения.

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#71] [Административная РГ]
Для более точного понимания смысла полей при их моделировании в FHIR хотелось бы иметь пример правильного заполнения. Может ли кто-то дать такой пример? 
Это могла бы быть фактическая строка из справочника либо фактическая строка с исправленными полями, в случае, если у ЦНИИОИЗ есть претензии к фактическому заполнению.

**Comment by iradio** [comment]
Прилагается деперсонифицированный пример данных (ИНН, СНИЛС и паспортные данные заменены синтетическими)  для организации "Медицинский радиологический научный центр имени А.Ф. Цыба - филиал федерального государственного бюджетного учреждения «Национальный медицинский исследовательский центр радиологии» Министерства здравоохранения Российской Федерации" из разрабатываемой сейчас версии ФРМО.
[DD_Radiologiya_eventprocessor_logs1.zip](https://github.com/fhir-ru/core/files/8160854/DD_Radiologiya_eventprocessor_logs1.zip)

**Reply by Vrodvi** [reply]
1) Коллеги, проверьте, пожалуйста, корректность заполнения присланного пример. 
Многие поля установлены в значения null или 0.
Плюс вижу, что у данной организации указан тип `Государственное юридическое лицо`, хотя это филиал:
```
        "OrgType": {
            "Code": 1,
            "Name": "Государственное юридическое лицо"
```
Где в данных найти, что это именно филиал, и где указана связь с родительским юридическим лицом?
Прошу проверить правильность выгрузки данных.

2) Также хочу отметить, что исходно запрашивался (см. https://github.com/fhir-ru/core/discussions/26#discussioncomment-1693972) полный пример государственной организации со сложной организационной структурой, чтобы понять каким образом эта структура отображается сейчас в ФРМО. Поскольку предоставлен пример только одного филиала организации, то это не очень поможет проанализировать принцип описания структуры.
Можно выгрузить полный пример для организации и `ФГБУ "НМИЦ РАДИОЛОГИИ" МИНЗДРАВА РОССИИ` со всеми её филиалами?
А ещё лучше взять для примера организацию `"ГБУЗ МКНЦ имени А.С. Логинова ДЗМ", ОГРН 1027700189626` (см. https://github.com/fhir-ru/core/discussions/26#discussioncomment-1693435) - в её структуре, например, имеются филиалы у филиалов (не юридически, но организационно) - надо понимать каким образом эта сложная структура сейчас описана в ФРМО.

3) Зачем в этом файле 97611 строк, из которых большая часть - это данные о сотрудниках, хотя запрашивался пример из ФРМО, а не из ФРМР? Для примера будет достаточно пары человек.

4) Прошу вместе с примером публиковать ещё и описание структуры\атрибутивный состав, т.к. просто из английских названий атрибутов не совсем понятно их предназначение, например:
```
		"Site": null,
		"AttachedPeople": 0,
		"MedicalInformationSystem": false,
		"RoutingOKS": null,
		"RoutingNumber": 0,
		"PrimaryVascular": false,
```

5) Там где в структуре примера указаны кодированные данные - как понять по какому справочнику осуществлено кодирование? Например:
```
		"OrgType": {
			"Code": 1,
			"Name": "Государственное юридическое лицо"
		},
```
